SPCTR-3327: Fix - Core Social share alignment issue inside container. (#3530)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Global - Resolved an issue in which the font-family that was set would not appear as intended in the FSE template editor.
* Fix: Separator - Resolved an issue wherein the height would revert from percentage to pixels after reloading the editor.
* Improvement: Global - Enhanced the reusable block functionality for the following blocks: Tabs and Slider.

> * Fix: Container - Resolved an issue wherein the core social share block is getting center aligned by default inside Container block.

Reviewer: ```suggestion
* Fix: Container - Resolved an issue wherein the core Social Share block would get centered by default inside Containers.
```

Author: Done.
